src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -8,61 +8,126 @@ translate:
+New Relic offers several methods to collect performance data about your AWS environment. Once you've connected New Relic with AWS, you get total visibility into AWS telemetry data, letting you closely monitor the health of your AWS stack. 
+
+## Integrate New Relic with your AWS stack [#connect]
+
+To get started, you need to establish a connection between New Relic and your AWS services. Establishing the initial connection integrates New Relic into your AWS stack so you can access additional integrations for full visibility into your AWS environment. 
+
+Here are options to get started:
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                AWS solution
+            </th>
+            <th>
+                New Relic integration method
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                CloudWatch metric streams
+            </td>
+            <td>
+                [Set up Cloudwatch Metric Streams integration]() (Recommended)
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Amazon EC2 instances
+            </td>
+            <td>
+                [Install the infrastructure agent with CloudFormation template]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                AWS GovCloud (US)
+            </td>
+            <td>
+                [Connect your Amazon account to New Relic]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                AWS System Manager Distributor
+            </td>
+            <td>
+                [Install the infrastructure agent on AWS System Manager Distributor]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Amazon Elastic Kubernetes Service (Amazon EKS)
+            </td>
+            <td>
+                [Add New Relic to a custom resource definition (CRD) file]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Connect to the AWS Polling API
+            </td>
+            <td>
+                [Create new roles and policies to poll and send metric data to New Relic]()
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## View your AWS data [#view-aws-data]
+
+Once installed and configured, you can access data from your AWS services directly in the New Relic platform. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Amazon Web Services**</DNT>. You can start building queries from <DNT>**Metrics & Events**</DNT> as well, or by interacting with our query tool anchored at the bottom of your browser.
+
+## Amazon CloudWatch Metric streams and API polling [#api-polling]
+
+We strongly recommend that you use the CloudWatch Metric Streams integration as it directly connects with the metric streams repository that holds your AWS metric data. You may need our API Polling integrations to supplement AWS CloudWatch metric streams. 
```

Comment:
@akristen strongly recommend this instead of....? Also, should we link to the CloudWatch integration doc here?

Reply:
I'm a little hazy on the details, but CloudWatch Metric Streams is an o11y mechanism for collecting metric data from AWS integrations. It replaces API polling in almost all cases, but there are some edge situations where API polling is necessary. I'll follow up with a SME about this too because the docs in general could use more clarity about this